app/plugins/Awards/src/Services/BestowalFinalizationService.php

```
@@ -176,13 +211,53 @@ protected function applyGiven(Bestowal $bestowal, int $actorId, ?DateTimeInterfa
      * Load a bestowal by id, returning null when it does not exist.
      *
      * @param int $bestowalId Bestowal id.
+     * @param bool $forUpdate Whether to acquire a row lock.
      * @return \Awards\Model\Entity\Bestowal|null
      */
-    protected function loadBestowal(int $bestowalId): ?Bestowal
+    protected function loadBestowal(int $bestowalId, bool $forUpdate = false): ?Bestowal
     {
         /** @var \Awards\Model\Entity\Bestowal|null $bestowal */
-        $bestowal = $this->bestowals->find()->where(['Bestowals.id' => $bestowalId])->first();
+        $query = $this->bestowals->find()->where(['Bestowals.id' => $bestowalId]);
```

PHPStan (2.2.7)

[error] Variable $bestowal in PHPDoc tag `@var` does not match assigned variable $query.


(varTag.differentVariable)